src/styles/TabViewStyle.qml

```diff
@@ -20,6 +20,32 @@
 import QtQuick 2.6
 import QtQuick.Controls.Styles 1.0
 
-TabViewStyle { 
```

Comment:
@neochapay's changes in his PR are needed here.
https://github.com/nemomobile/qtquickcontrols-nemo/pull/50